.github/workflows/scanbuild.yml

```diff
@@ -48,10 +48,7 @@ jobs:
                 CC: ${{ matrix.compiler }}
                 MAKEVARS: ${{ matrix.makevars }}
               run: |
-                autoreconf -i
```

Comment:
Oops

Reply:
Should be fixed with https://github.com/jqlang/jq/commit/54fef09ac39d6fb003367d65709349c43078405e

Reply:
Thank you for fixing it @emanuele6!